app-emulation/firecracker-bin/firecracker-bin-0.23.0.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-info
+
+DESCRIPTION="Secure and fast microVMs for serverless computing (static build)"
+HOMEPAGE="https://firecracker-microvm.github.io https://github.com/firecracker-microvm/firecracker"
+SRC_URI="
+	amd64? (
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/firecracker-v${PV}-x86_64
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/jailer-v${PV}-x86_64
+	)
+	arm64? (
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/firecracker-v${PV}-aarch64
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/jailer-v${PV}-aarch64
+	)"
+
+LICENSE="|| ( Apache-2.0 MIT Apache-2.0-with-LLVM-exceptions ) MPL-2.0"
+SLOT="0"
+KEYWORDS="-alpha ~amd64 -arm -hppa -ia64 -ppc -ppc64 -sparc -x86"
```

Comment:
This is pretty useless, it's the same as leaving it null. Seeing this would confuse people, let's just take all `-` KEYWORDS out.

Reply:
Really? The [dev manual](https://devmanual.gentoo.org/keywording/index.html) says that not mentioning an arch means that support is unknown, while `-arch` means that the arch is known to be unsupported. I think the latter is the case here, since upstream releases binaries only for two arches (`amd64` and `arm64`). Binary only packages are explicitly mentioned as an example for when `-arch` is sensible.
I really just "expanded" the `-*` (as used in `app-emulation/firecracker-bin-0.22.1`) to be able to make `arm64` "unknown", so that hopefully that keyword can be added. This was suggested by @thesamesam in https://bugs.gentoo.org/740886#3.
Did I mix something up here?

Reply:
In both cases it should be masked by a missing keyword, and be unfetchable anyway. so it doesn't make any difference. This is just ugly.

Reply:
Alright, changed it.